pkg/deploy/elbv2/listener_manager.go

```diff
@@ -354,3 +366,16 @@ func buildResListenerStatus(sdkLS ListenerWithTags) elbv2model.ListenerStatus {
 		ListenerARN: awssdk.ToString(sdkLS.Listener.ListenerArn),
 	}
 }
+
+var PROTOCOLS_WITH_ATTRIBUTES = map[elbv2model.Protocol]bool{
```

Comment:
Nit:
1. Define these constants at the beginning 
2. Change from `var` to `const`
3. Change constant name to reflect listener attributes `PROTOCOLS_SUPPORTING_LISTENER_ATTRIBUTES`

Reply:
Resolved 1 and 3. 

1. Defined it at the beginning
3. Changed constant name to reflect listener attributes PROTOCOLS_SUPPORTING_LISTENER_ATTRIBUTES.

However, for 2. I can't set it as const because maps cannot be declared as constants in Go.